src/main/java/core/basesyntax/Report.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+public class Report {
```

Comment:
for this task you don't need to create such a class, remove it and rewrite your logic, it's enough to create a separate method that creates report inside of WorkWithFile